### PR TITLE
LPJ content update for data source change

### PR DESCRIPTION
**Pull request by paridhi-parajuli**
## Why are you creating this Pull Request?

- Minor content changes to [LPJ data](https://docs.google.com/document/d/1RrzHXg2GA2NLZB0E5SidK2t6pvKnFhKp_23X_Fiq-lY/edit)

### CODE REVIEW

**Comment by slesaad** [comment]
@paridhi-parajuli can you change the rescale to `0` and `3*e-9`

**Comment by Jeanne-le-Roux** [comment]
There may be a few additional changes to add, FYI

**Comment by paridhi-parajuli** [comment]
@Jeanne-le-Roux I have added the additional content changes.

File: datasets/lpjwsl-wetlandch4-grid-v2.data.mdx [hunk 1]
```diff
@@ -58,7 +58,7 @@ layers:
       colormap_name: magma
       rescale:
         - 0
-        - 3
+        - 3*e-9
```
Comment (slesaad) [inline datasets/lpjwsl-wetlandch4-grid-v2.data.mdx:61]:
```suggestion
        - 3e-9
```

Comment (slesaad) [reply datasets/lpjwsl-wetlandch4-grid-v2.data.mdx:61]: * doesn't work apparently

Comment (slesaad) [reply datasets/lpjwsl-wetlandch4-grid-v2.data.mdx:61]: change this everywhere

**Comment by slesaad** [comment]
@paridhi-parajuli fix the merge conflicts and mark this ready to review

**Review by slesaad — APPROVED** [review]

File: datasets/lpjwsl-wetlandch4-grid-v2.data.mdx [hunk 2]
```diff
@@ -68,7 +68,7 @@ layers:
         }
     legend:
       unit:
-        label: g CH₄/m²/month
+        label: kg CH₄/m²/s
       type: gradient
       min: 0
       max: 3
```
Comment (slesaad) [inline datasets/lpjwsl-wetlandch4-grid-v2.data.mdx:74]: 👀

**Pull request merged**